test/utils/axe-helper.js

```diff
@@ -16,7 +16,7 @@ exports.getAccessibilityTestResults = (componentName) => {
         const options = {
             runOnly: {
                 type: 'tag',
-                values: ['wcag21aa']
+                values: ['wcag21a', 'wcag21aa', 'wcag143', 'cat.color', 'cat.aria']
```

Comment:
Out of interest, where are these defined? Is there documentation that could be linked?

Reply:
So these are defined by axe-core and deque. But that reminds me, maybe I should add some documentation on this into the storybook files. I'll add that now as part of this PR.

https://github.com/dequelabs/axe-core/blob/e699939bfd43fcc66b357d0e7329adce6f29cd6b/doc/API.md#axe-core-tags